schemas/services.role.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+var constants = require('bedrock').config.constants;
+var schemas = require('bedrock-validation').schemas;
+
+var postRole = {
+  type: 'object',
+  properties: {
+    comment: schemas.description({required: false}),
+    label: schemas.label({required: false}),
+    sysPermission: {
+      type: 'array',
+      required: true,
+      minItems: 1,
+      uniqueItems: true,
+      items: {
+        type: 'string'
+      }
+    }
+  },
+  additionalProperties: false
+};
+
+var patchRole = {
+  type: 'object',
+  properties: {
+    id: schemas.identifier(),
+    comment: schemas.description({required: false}),
+    label: schemas.label({required: false}),
+    sysPermission: {
+      type: 'array',
+      required: true,
+      minItems: 1,
+      uniqueItems: true,
+      items: {
+        type: 'string'
+      }
+    }
```

Comment:
This schema doesn't look right. A patch should be a list of commands to be executed not a single object for a single update.